pyomo/util/report_scaling.py

```diff
@@ -0,0 +1,201 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+import pyomo.environ as pe
+import math
+from pyomo.core.base.block import _BlockData
+from pyomo.common.collections import ComponentSet
+from pyomo.core.base.var import _GeneralVarData
+from pyomo.contrib.fbbt.fbbt import compute_bounds_on_expr
+from pyomo.core.expr.calculus.diff_with_pyomo import reverse_sd
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def _get_longest_name(comps):
+    longest_name = 0
+
+    for i in comps:
+        i_len = len(str(i))
+        if i_len > longest_name:
+            longest_name = i_len
+
+    if longest_name > 195:
+        longest_name = 195
+    if longest_name < 12:
+        longest_name = 12
+
+    return longest_name
+
+
+def _print_var_set(var_set):
+    longest_varname = _get_longest_name(var_set)
+
+    s = f'{"Var":<{longest_varname + 5}}{"LB":>12}{"UB":>12}\n'
+
+    for v in var_set:
+        if v.lb is None:
+            v_lb = -math.inf
+        else:
+            v_lb = v.lb
+        if v.ub is None:
+            v_ub = math.inf
+        else:
+            v_ub = v.ub
```

Comment:
We repeat this a LOT.  Would a helper function be warranted:
```python
def _bounds_to_float(lb, ub):
    if lb is None:
        lb = -math.inf
    if ub is None:
        ub - math.inf
    return lb, ub
```
then, e.g.
```python
for c in var_set:
    v_lb, v_ub = _bounds_to_float(*v.bounds)
```

Reply:
I like it.